### DISCUSSION TITLE
Send from Wallet using BTCPayServer.Client Nuget

### DISCUSSION THREAD

**Discussion by zachclements** [btcpayserver/btcpayserver#3174] [Technical Support] [answered]
I want to be able to send from my hot wallet in BTC PayServer using the c# client. Does anyone know if it is supported and if so how to do it?

**Comment by zachclements** [comment, marked as answer]
I figured it out for anyone else here is a way to do it. probably not the best coding but it works.

`            
string storeId = "insert store id here";
BTCPayServer.Client.BTCPayServerClient client = new BTCPayServer.Client.BTCPayServerClient(new Uri("https://btcpayserver.url/"), "username@domain.com", "passsword");
            BTCPayServer.Client.Models.CreateOnChainTransactionRequest trx = new CreateOnChainTransactionRequest();
            CreateOnChainTransactionRequest.CreateOnChainTransactionRequestDestination dest = new CreateOnChainTransactionRequest.CreateOnChainTransactionRequestDestination();
            dest.Amount = 0.00020356m;
            dest.Destination = "[DESTINATION ADDRESS]";
            dest.SubtractFromAmount = false;
            trx.Destinations = new List<CreateOnChainTransactionRequest.CreateOnChainTransactionRequestDestination>();
            trx.Destinations.Add(dest);
            trx.ProceedWithBroadcast = true;
            var results = client.CreateOnChainTransaction(storeId, "BTC", trx).GetAwaiter().GetResult();`

**Comment by Clayodell1989** [comment]
Ty
